# miRNA-analysis collaboration with Dr. Christain, University of Barcelona, Spain. Date created/updated: December, 9, 2024. https://www.ub.edu/farmaco/en/pharmacognosy/research/neuropharmacology_in_aging_and_neurodegeneration/6/
<img width="431" alt="Data brief" src="https://github.com/spawar2/miRNA-analysis/assets/25118302/516c18c6-2c87-45ec-9d7b-1293085a7bdd">

[R: DESeq2, tidyverse].
Elsevier Data in Brief. [Paper Link:](https://www.sciencedirect.com/science/article/pii/S235234092100398X)
<img width="579" alt="Screenshot 2023-05-17 at 10 30 50 AM" src="https://github.com/spawar2/miRNA-analysis/assets/25118302/acbc7f91-f3da-4286-a4d0-aed62e716285">
Georgia State University, Atlanta, United States of America (USA).
https://csds.gsu.edu/ Digital Object Identifier (DOI)=Online International Standard Book Number (ISBN).
RNA-seq and miRNA-seq data from pharmacological inhibition of the G9a/GLP histone methyltransferase complex with UNC0642 in SAMP8 mice, Shrikant Pawar, AinaBellver-Sanchis, ChristianGriñán-Ferré, Publication: Elsevier Data in Brief (Publication date: May 3), collaboration with Dr. ChristianGriñán-Ferré, University of Barcelona, Spain, IF=1.38, Github, Article link ^^^ DOI: https://doi.org/10.1016/j.dib.2021.107114, Volume: 36, Issue, Pages: 107114.
deseq.R: MiRNAMicro ribonucleic acid data sequencing headcount read, DESeq differential expression analysis, Mean-Average MA Plotting the expression, Kegg enrichment analysis.
selected function(as.data.frame, DESeqDataSetFromMatrix, results, tbl_df, sizeFactors, dispersions, names, DESeq).
††^^^Georgia State University, College of Arts and Sciences, Department of Computer Science and Biology, Atlanta, United States of America (USA). 
†Corresponding author. ††First author. †††Second author. ††††Third author. †††††author.

https://www.java.com/
https://www.php.net/
https://www.mysql.com/
https://www.mathworks.com/products/matlab.html

https://www.r-project.org/
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html
https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html
https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html
https://ggplot2.tidyverse.org/
https://cran.r-project.org/package=randomForest
https://topepo.github.io/caret/
https://bioconductor.org/packages/release/bioc/html/DESeq2.html
https://cran.r-project.org/package=missForest
https://bioconductor.org/packages/release/bioc/html/affy.html
https://bioconductor.org/packages/release/bioc/html/limma.html
https://rstudio.github.io/DT/
https://bioconductor.org/packages/release/bioc/html/edgeR.html
https://apps.cytoscape.org/apps/cytohubba
https://cytoscape.org/
https://string-db.org/
https://www.gsea-msigdb.org/
https://bioconductor.org/packages//2.13/bioc/html/BGmix.html
https://ggplot2.tidyverse.org/
https://www.htmlwidgets.org/
https://cran.r-project.org/package=cluster
https://cran.r-project.org/package=fpc
https://cran.r-project.org/package=survival
https://www.bioconductor.org/packages/release/bioc/html/multiClust.html
https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html
https://cran.r-project.org/package=dendextend
https://cran.r-project.org/package=amap
https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://shiny.posit.co/
https://cran.r-project.org/package=ggbeeswarm
https://cran.r-project.org/package=speedglm
https://magrittr.tidyverse.org/

https://www.python.org/
https://scikit-learn.org/
https://pandas.pydata.org/
https://matplotlib.org/
https://www.tidyverse.org/
https://numpy.org/
https://www.tensorflow.org/
https://kivy.org/
https://pydicom.github.io/
https://docs.python.org/3/library/shlex.html
https://scipy.org/
https://scikit-image.org/
https://seaborn.pydata.org/
https://librosa.org/doc/
https://docs.python.org/3/library/glob.html
https://pytorch.org/
https://pydicom.github.io/
https://biopython.org/docs/dev/api/Bio.Blast.html
https://pypi.org/project/openai/0.26.5/
https://docs.python.org/3/library/tkinter.html
https://docs.fast.ai/
https://pypi.org/project/torchvision/
